### DISCUSSION TITLE
PCL为什么不添加一个类似hmcl那样的跳过账户刷新的按钮呢？

### DISCUSSION THREAD

**Discussion by 4964368** [Meloong-Git/PCL#3542] [提问] [answered]
感觉那个功能会有很多人能用的到（

**Comment by allMagicNB** [comment]
哪个账户刷新？

**Reply by 4964368** [reply]
就是那个未能访问到登陆服务器那里有个跳过账户刷新直接启动游戏

**Comment by MoYuan-CN** [comment]
未找到相关 Issue（检索式：`is:issue 跳过刷新 跳过正版刷新 `）
开个新的 Issue？

**Reply by allMagicNB** [reply]
有出现过这种错误吗……

**Reply by 4964368** [reply]
我这应该算是建议……吧？

**Reply by Deep-Dark-Forest** [reply]
> 我这应该算是建议……吧？

Issue不是提建议用的吗

**Reply by iPodtouch6557** [reply]
新功能模板欢迎你，快去吧

**Comment by MoYuan-CN** [comment]
后续讨论移步 #3543

**Reply by allMagicNB** [reply]
没解答呢，别乱 Mark as answer

**Comment by allMagicNB** [comment, marked as answer]
这个实现的原理实际上就是连不上微软时仍使用 UUID 与用户名啥的来直接进行启动……
每次启动游戏时，启动器都会发送一个请求，根据你的登陆信息向微软请求 accessToken，这个 Token 会过期，HMCL 是会使用上一次成功获取的 Token 启动游戏，隔了太长时间 Token 也会失效
所以
#1660（#1664）
